extensions/Cake.Deploy.Azure.ResourceManager.yml

```diff
@@ -8,7 +8,7 @@ Author: ObjectivityLtd
 Description: Cake addin to use Azure Resource Manager template.
 Categories:
 - azure
-- resource-manager
```

Comment:
@Jericho Is that not valid from addin discoverer point of view?

Reply:
Removing dashes is logic that was added in [AddinDisco 3.31.0 released in August 2019](https://github.com/cake-contrib/Cake.AddinDiscoverer/compare/3.30.0...3.31.0). I'm pretty sure at that time Cake's web site couldn't handle dashes in addin categories which is why, after discussing with @gep13, we decided to remove them.

Reply:
It seems to work fine in current website (https://cakebuild.net/extensions/cake-deploy-azure-resourcemanager/). We can also go with spaces, but using a dash generally shound like a less troublesome solution and also common on other sites.

Reply:
ok no problem. I'll change the logic so we preserve dashes.